python/cugraph/cugraph/tests/data_store/test_property_graph.py

```diff
@@ -1176,10 +1176,12 @@ def test_extract_subgraph_vertex_prop_condition_only(
         )
     # Should result in two edges, one a "relationship", the other a "referral"
     expected_edgelist = cudf.DataFrame(
-        {"src": [89216, 78634], "dst": [78634, 89216], "weights": [99, 8]}
+        {"src": [78634, 89216], "dst": [89216, 78634], "weights": [8, 99]}
```

Comment:
Wonder why we flipped it around ?

Reply:
I initially removed the `drop_duplicate` call which was swapping the order of some columns. This expected result was set to match the actual_result. But this is no longer relevant since I am no longer updating this test file

Reply:
Thanks for fixing this.